docs/docs/05-configuration.md

```diff
@@ -31,7 +31,7 @@ $ snowpack dev --no-bundle
 
 - **`--config [path]`** Set the path to your project config file.
 - **`--help`** Show this help.
-- **`--version`** Show the current version. 
+- **`--version`** Show the current version.
```

Comment:
My editor is configured to automatically strip trailing whitespace. I can get rid of the extra changes if needed.

Reply:
This is fine! TBH I’ve had the same issue myself. Thanks for fixing.